TestProjects/VisualEffectGraph_URP/ProjectSettings/ProjectSettings.asset

```diff
@@ -576,19 +576,19 @@ PlayerSettings:
   webGLThreadsSupport: 0
   webGLDecompressionFallback: 0
   scriptingDefineSymbols:
-    1: UNITY_POST_PROCESSING_STACK_V2;URP_TEST_AGGRESSIVE_SHADER_STRIPPING;URP_ENABLE_DEBUG_DISPLAY
-    4: UNITY_POST_PROCESSING_STACK_V2;URP_TEST_AGGRESSIVE_SHADER_STRIPPING
-    7: UNITY_POST_PROCESSING_STACK_V2;URP_TEST_AGGRESSIVE_SHADER_STRIPPING
-    13: UNITY_POST_PROCESSING_STACK_V2;URP_TEST_AGGRESSIVE_SHADER_STRIPPING
-    14: UNITY_POST_PROCESSING_STACK_V2;URP_TEST_AGGRESSIVE_SHADER_STRIPPING
-    18: UNITY_POST_PROCESSING_STACK_V2;URP_TEST_AGGRESSIVE_SHADER_STRIPPING
-    19: UNITY_POST_PROCESSING_STACK_V2;URP_TEST_AGGRESSIVE_SHADER_STRIPPING
-    21: UNITY_POST_PROCESSING_STACK_V2;URP_TEST_AGGRESSIVE_SHADER_STRIPPING
-    23: UNITY_POST_PROCESSING_STACK_V2;URP_TEST_AGGRESSIVE_SHADER_STRIPPING
-    25: UNITY_POST_PROCESSING_STACK_V2;URP_TEST_AGGRESSIVE_SHADER_STRIPPING
-    26: UNITY_POST_PROCESSING_STACK_V2;URP_TEST_AGGRESSIVE_SHADER_STRIPPING
-    27: UNITY_POST_PROCESSING_STACK_V2;URP_TEST_AGGRESSIVE_SHADER_STRIPPING
-    28: UNITY_POST_PROCESSING_STACK_V2;URP_TEST_AGGRESSIVE_SHADER_STRIPPING
+    1: UNITY_POST_PROCESSING_STACK_V2
+    4: UNITY_POST_PROCESSING_STACK_V2
```

Comment:
I don't understand wher this define come from (UNITY_POST_PROCESSING_STACK_V2), we don't have PPV2 in our test project AFAIK

Reply:
I didn't know what it is, and that's why I preferred not to touch it...